reana.yaml

```diff
@@ -11,10 +11,14 @@ workflow:
   type: serial
   specification:
     steps:
-      - environment: 'reanahub/reana-env-root6'
+      - name: gendata
+        environment: 'reanahub/reana-env-root6'
         commands:
         - mkdir -p results
         - root -b -q 'code/gendata.C(${events},"${data}")' | tee gendata.log
+      - name: fitdata
```

Comment:
You can do the same for `reana-htcondorcern.yaml` as part of the same commit, so that both K8s and HTC workflows stay basically the same. 

(I.e. if we introduce step names to Serial to some demo example, we could do it for all Serial workflows in that demo example.)